### ISSUE TITLE
Docs don't give examples of all supported constructor arguments

### ISSUE THREAD

**Issue by thw0rted** [Qix-/color#181]
The docs give examples of various forms of `Color.rgb()` with 3 arguments, but never give an example of specifying an alpha value in the constructor.  They do give the example `Color('#7743CE').alpha(0.5)` but nothing like `Color.rgb([0x77, 0x43, 0xCE, 0.5])`.

Also, there are no examples of construction from non-RGB color spaces.  `Color.hsl(200,80,70)` works but is never mentioned.

**Comment by Qix-** [comment]
Yeah, the readme needs a makeover for sure. Feel free to open a PR.
